src/components/Profile/components/PersonalInfoList/components/UpdatePhoneDialog/index.tsx

```diff
@@ -67,20 +74,22 @@ const UpdatePhone = () => {
 
 // From material ui website
 // https://material-ui.com/components/text-fields/#integration-with-3rd-party-input-libraries
-const phoneMaskUS = forwardRef((props, ref) => {
-  const { onChange, ...other } = props;
+const phoneMaskUS = forwardRef(
+  (props: { onChange: ({}) => void; name?: string } & InputBaseComponentProps, ref) => {
```

Comment:
This is an incorrect type for `onChange` - please specify the type more accurately.